src/model/entities/rundown.ts

```diff
@@ -495,6 +496,10 @@ export class Rundown extends BasicRundown {
     const nextSegment: Segment = this.findSegment(segmentId)
     const nextPart: Part = nextSegment.findPart(partId)
 
+    if (nextSegment.invalidity) {
+      throw new InvalidSegmentException(`Unable to set segment ${nextSegment.name} as next, since it is invalid.`)
```

Comment:
Should `${nextSegment.name}` be in quotation marks?

Reply:
Sure. It has been updated.